cluster-autoscaler/Makefile

```diff
@@ -1,5 +1,5 @@
 ALL_ARCH = amd64 arm64 s390x
-all: $(addprefix build-arch-,$(ALL_ARCH))
+all: $(addprefix build-arch-,$(ALL_ARCH)) $(addprefix make-image-arch-,$(ALL_ARCH))
```

Comment:
Can we split the new stuff into a new target? Maybe `all-images`? The Cloud Build process has a 1 hour timeout and is pretty slow so whatever we can do to trim it down would be good.

Reply:
would `all-images` include build-arch + make-image-arch for all architectures?